api/PeriodicWave.json

```diff
@@ -29,10 +29,10 @@
             "version_added": "14"
           },
           "safari": {
-            "version_added": "6"
+            "version_added": "7"
```

Comment:
The original 6 makes more sense that's when `createPeriodicWave` and `OscillatorNode` introduced. Was this a case of the interface object not being exposed?

Reply:
Oh, oops... I could have sworn there was a custom test for this, my bad!

Reply:
We have custom tests for lots of audio nodes, so it's easy to get them mixed up :)